.travis.yml

```diff
@@ -10,7 +10,9 @@ matrix:
     os: linux
     env: NODE_VERSION=v8.16.0
 language: go
-go: 1.12.9
+go:
```

Comment:
What does it mean to list two values here?

Reply:
This means we will run builds for each of the values in that matrix - this will ensure that we get better coverage of both 1.12.x and 1.13